src/annotator/test/annotation-sync-test.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var AnnotationSync = require('../annotation-sync');
+var EventEmitter = require('tiny-emitter');
```

Comment:
I've made this change, but do we have a JavaScript style guide that we're following?